samples/Hardware.GiantGecko/GiantGecko.PowerMode/Program.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Device.Gpio;
+using System.Diagnostics;
+using System.Threading;
+using GiantGecko = nanoFramework.Hardware.GiantGecko;
+
+namespace PowerMode
+{
+    public class Program
+    {
+        private static GpioController _gpioController;
+
+        public static void Main()
+        {
+            _gpioController = new GpioController();
+
+            // Silabs SLSTK3701A: LED1 PH14 is LLED1
+            GpioPin led = _gpioController.OpenPin(PinNumber('H', 14), PinMode.Output);
+
+            // start a thread blinking the LED to check that something is happening 
+            new Thread(() => {
+                while (true)
+                {
+                    Thread.Sleep(125);
+                    led.Toggle();
+                }
+            }).Start();
+
+            // sleep here for 5 seconds to allow the LED to blink
+            Thread.Sleep(5_000);
+
+            Debug.WriteLine($"Going to standby mode now...");
+
+            //////////////////////////////////////
+            // *** these calls never return *** //
+            //////////////////////////////////////
+            
+            // after this the target will enter EM4 hibernate mode (RTC will be on and counting)
+            GiantGecko.Power.EnterHibernateMode();
```

Comment:
will the execution resumes here or not?
If not then I gues a comment to say use either this line or the other can help

Reply:
thanks for the adjustments, all good and clear now!